modules/admin_manual/pages/configuration/server/language_configuration.adoc

```diff
@@ -19,4 +16,7 @@ Please check settings/languageCodes.php for the list of supported language codes
   "default_language" => "en",
 ....
 
-This parameters can be set in the config/config.php
+Please check _settings/languageCodes.php_ for the list of supported language codes.
```

Comment:
hmmm, if I list the settings folder of my productive instance I see the following:
```
drwxr-x--- 12 root www-data  4096 Mar 26 13:22 ./
drwxr-x--- 12 root www-data  4096 Apr  2 20:39 ../
drwxr-x---  2 root www-data  4096 Mar 26 13:22 ajax/
-rw-r-----  1 root www-data 10410 Mar 26 13:22 Application.php
drwxr-x---  2 root www-data  4096 Mar 26 13:22 ChangePassword/
drwxr-x---  2 root www-data  4096 Mar 26 13:22 Controller/
drwxr-x---  2 root www-data  4096 Mar 26 13:22 css/
-rw-r-----  1 root www-data  2033 Mar 26 13:22 help.php
drwxr-x---  2 root www-data  4096 Mar 26 13:22 img/
drwxr-x---  4 root www-data  4096 Mar 26 13:22 js/
drwxr-x---  2 root www-data 16384 Mar 26 13:22 l10n/
drwxr-x---  2 root www-data  4096 Mar 26 13:22 Middleware/
drwxr-x---  4 root www-data  4096 Mar 26 13:22 Panels/
-rw-r-----  1 root www-data  6769 Mar 26 13:22 routes.php
drwxr-x---  5 root www-data  4096 Mar 26 13:22 templates/
-rw-r-----  1 root www-data  4993 Mar 26 13:22 users.php
```

Reply:
Good catch. I will check this!

Reply:
pls also check how the listed content will result to a language code used

Reply:
https://github.com/owncloud/core/blob/v9.1.8/settings/languageCodes.php
Seems like this page is deprecated in ownCloud 10